beacon_node/beacon_chain/src/fetch_blobs.rs

```diff
@@ -180,7 +177,7 @@ async fn fetch_and_process_blobs_v1<T: BeaconChainTypes>(
         .process_engine_blobs(
             block.slot(),
             block_root,
-            EngineGetBlobsOutput::Blobs(fixed_blob_sidecar_list.clone()),
+            EngineGetBlobsOutput::Blobs(blobs_to_import_and_publish),
```

Comment:
Bug fix: we previously pass all blobs instead of `blobs_to_import_and_publish` to processing.